docs/source/getting_started/installation.md

```diff
@@ -87,3 +104,46 @@ Known labextensions:
         @jupyterlab/toc v4.0.0  enabled  OK
         nbdime-jupyterlab v2.0.0  enabled  OK
 ```
+
+### Docker 
+
+If you have Docker installed, you can use JupyterLab and Elyra by running one of the [ready-to-run Docker images](https://hub.docker.com/r/elyra/elyra/tags) maintained by the Elyra Team:
+
+ - `elyra/elyra:latest` has the latest released version installed.
+ - `elyra/elyra:x.y.z` has version `x.y.z` installed.
+ - `elyra/elyra:dev` is automatically re-built each time a change is committed to the master branch.
+
+Invocation example 1: Run the most recent Elyra development build in a Docker container. All changes are discarded when the Docker container is stopped.
+
+```
+docker run -it -p 8888:8888 elyra/elyra:dev jupyter lab --debug
+```
+
+Invocation example 2: Run the most recent Elyra development build in a Docker container and mount the existing local `$HOME/jupyter-notebooks/` directory as JupyterLab work directory. This enables you to make existing notebooks and other files available in the Docker container. Only files in this working directory are retained when the Docker container is stopped. 
+
+```
+docker run -it -p 8888:8888\
+ -v ${HOME}/jupyter-notebooks/:/home/jovyan/work\
+ -w /home/jovyan/work\
+ elyra/elyra:dev jupyter lab --debug
+```
+
+Invocation example 3: Same as above. In addition the existing local `$HOME/jupyter-data-dir` directory is mounted as the Jupyter data directory, preserving all user-defined Elyra metadata artifacts you create, such as code snippets, runtime configurations, or runtime images.
```

Comment:
Just to make sure, there is a command `jupyter --data-dir` that gives you the location of the data directory (e.g. `/Users/lresende/Library/Jupyter` in my macos dev environment`

Reply:
It turns out we were thinking about two different scenarios and my arbitrary directory name choice was ambiguous. For the purpose of this documentation `jupyter-data-dir` is a placeholder for any valid local directory name.
 - If `$HOME/jupyter-data-dir`  refers to an empty local directory JupyterLab in the Docker container is started in a "clean" environment. 
 - If `$HOME/jupyter-data-dir`  refers to the data directory of a local JupyterLab installation (`jupyter --data-dir`) then the Jupyter metadata is shared between the local instance and the instance in the Docker container. This is an advanced usage scenario.

I updated the document accordingly, which should hopefully address the concern.